examples/default-theme/src/App.vue

```diff
@@ -0,0 +1,134 @@
+  <div id="root">
+    <ais-index
+      :search-client="searchClient"
+      index-name="instant_search"
+      :routing="true"
+    >
+      <ais-configure :hitsPerPage="16" />
+
+      <header class="header">
+        <img class="logo" src="https://res.cloudinary.com/hilnmyskv/image/upload/w_100,h_100,dpr_2.0//v1461180087/logo-instantsearchjs-avatar.png" width="40">
+        <ais-search-box placeholder="Search a product" />
+      </header>
+
+      <main>
+        <aside>
+          <ais-clear-refinements>
+            <template slot="resetLabel">Clear all filters</template>
+          </ais-clear-refinements>
+
+          <ais-panel>
+            <template slot="header">Categories</template>
+            <ais-hierarchical-menu
+              :attributes="[
+                'hierarchicalCategories.lvl0',
+                'hierarchicalCategories.lvl1',
+                'hierarchicalCategories.lvl2',
+              ]"
+            />
+          </ais-panel>
+
+          <ais-panel>
+            <template slot="header">Brands</template>
+            <ais-refinement-list attribute="brand" />
+          </ais-panel>
+
+          <ais-panel>
+            <template slot="header">Rating</template>
+            <ais-rating-menu attribute="rating" :max="5" />
+          </ais-panel>
+
+          <ais-panel>
+            <template slot="header">Price</template>
+            <ais-range-input attribute="price" />
+          </ais-panel>
+
+          <div class="thank-you">
+            Data courtesy of <a href="https://developer.bestbuy.com/">Best Buy</a>
+          </div>
+        </aside>
+
+        <section>
+          <header class="section-header">
+            <ais-stats />
+            <div class="sort-by">
+              <label>Sort by</label>
+              <ais-sort-by
+                :items="[
+                  { name: 'instant_search', label: 'Featured' },
+                  { name: 'instant_search_price_asc', label: 'Price asc.' },
+                  { name: 'instant_search_price_desc', label: 'Price desc.' },
+                ]"
+              />
+            </div>
+          </header>
+
+          <ais-search-state>
+            <template slot-scope="{ query, hits }">
+              <p class="no-results" v-if="hits.length === 0">
+                No results found matching <em>{{ query }}</em>.
+              </p>
+            </template>
+          </ais-search-state>
+
+          <ais-hits>
+            <article
+              slot="item"
+              slot-scope="{ item }"
+              class="hit"
+            >
+              <div class="product-desc-wrapper">
+                <div class="product-name">
+                  <ais-highlight attribute="name" :hit="item" />
+                </div>
+                <div class="product-type">
+                  <ais-highlight attribute="type" :hit="item" />
+                </div>
+                <div class="product-footer">
+                  <div class="ais-RatingMenu-link">
```

Comment:
I wonder if we should expose this star rating thing somehow? probably out of scope for now though!